modules/docs/package.json

```diff
@@ -28,6 +28,7 @@
   "scripts": {
     "watch": "yarn build:es6 -w",
     "clean": "rimraf dist && rimraf ts3.5 && rimraf .build-info && mkdirp dist && mkdirp ts3.5/dist",
+    "build:mdx": "node ./utils/build.js mdx/",
```

Comment:
I couldn't suggest on the `build` script line, but currently nothing is calling this. Should this script be added to the `build` script below?